src/app/item-page/alerts/item-alerts.component.spec.ts

```diff
@@ -119,6 +119,7 @@ describe('ItemAlertsComponent', () => {
     tick();
     result$.subscribe((result) => {
       expect(result).toBeTrue();
+      done();
```

Comment:
The issue is still there, you can still change this to toBeFalse() and get the test to pass

Reply:
refactored